bin/ubuntu-core-initramfs

```diff
@@ -307,6 +307,17 @@ def create_initrd(parser, args):
             if os.path.exists(extra_modules):
                 add_modules_from_file(main, kernel_root, modules, firmware, extra_modules, db)
 
+        if "fips" in args.features:
+            subprocess.check_call(
+                [
+                    "/usr/lib/dracut/dracut-install",
+                    "-D", main,
+                    "--ldd", "--all",
+                    "/usr/bin/sha512hmac",
+                    "/usr/bin/.sha512hmac.hmac",
+                ] + glob.glob("/usr/lib/*/.libkcapi.so.*.hmac")
```

Comment:
I also cannot find this files in libkcapi1. Are these found only for package versions for fips maybe?

Reply:
Correct, only fips compliant version of kcapi-tools ships hmac files. In the same place as the fips compliant kernels.